cristin-import/src/test/java/no/unit/nva/cristin/lambda/CristinEntryEventConsumerTest.java

```diff
@@ -324,13 +332,92 @@ public void handleRequestThrowsExceptionWhenInputContainsUnknownProperty() throw
         assertThat(file, containsString(UNKNOWN_PROPERTY_NAME_IN_RESOURCE_FILE_WITH_UNKNOWN_PROPERTY));
     }
 
+    @Test
+    public void handlerThrowsExceptionWhenIsbnValueIsNull() {
+        CristinObject cristinInput = cristinDataGenerator.randomObject();
```

Comment:
I created a separate method creating a random object of type Monograph with bookreport values. But I do run into problems if I dont include the ".withBookReport()" when building the other versions of CristinObjects. This is because it fails the check assertThat(cristinObject, doesNotHaveEmptyValues()); which is included at the end of all the methods.